### ISSUE TITLE
Dropout application

### ISSUE THREAD

**Issue by ArlindKadra** [automl/Auto-PyTorch#188]
I have a question regarding dropout and how it is applied. In the previous non-refactored version of AutoPyTorch there was a `dropout_shape` hyperparameter, which does not seem to be there anymore. Is now dropout applied with the same rate at every layer and if not, does it follow the network shape then ?

**Comment by ravinkohli** [comment]
Hey, as far as I can tell, dropout_shape was not a hyperparameter but a parameter that depended on the network shape in the master branch. Similarly, it has exactly the same application in 'refactor_development'. You can find it [here](https://github.com/automl/Auto-PyTorch/blob/fae72a4de79b747918076292b0540a8e4602cb78/autoPyTorch/pipeline/components/setup/network_backbone/ShapedMLPBackbone.py#L46) and for reference, it is implemented using [this](https://github.com/automl/Auto-PyTorch/blob/fae72a4de79b747918076292b0540a8e4602cb78/autoPyTorch/pipeline/components/setup/network_backbone/utils.py#L133)
